controllers/k8ssandra/k8ssandracluster_controller.go

```diff
@@ -184,6 +184,13 @@ func (r *K8ssandraClusterReconciler) SetupWithManager(mgr ctrl.Manager, clusters
 		return requests
 	}
 
+	cb = cb.Watches(&source.Kind{Type: &cassdcapi.CassandraDatacenter{}},
```

Comment:
No need to watch the Medusa related resources here because it's handled by the other controller, correct?

Reply:
Correct